packages/geo/src/lib/import-export/import-export/import-export.component.html

```diff
@@ -141,6 +140,14 @@ <h4>{{'igo.geo.importExportForm.exportNoLayersExportable' | translate}}</h4>
     </mat-slide-toggle>
   </div>
 
+  <div class="export-options mat-typography" *ngIf="form.value.format !== 'URL' && circlesArePresent">
```

Comment:
This control is not useful. It is better if you directly store a polygon,

Reply:
I see... you store a circle and export a polygon.... to let the circle editable... ok...

Reply:
Yes, it was discussed with the team and it was determined that it was for the best to do it that way (give the user the option to transform in a polygon or keep the point with radius). If the circle is transformed in a polygon when the drawing is done, the circle is then a x sided polygon and you can't modify it. It really depends on the need of the user(s).

Reply:
This make no sense to let a point + a radius been exported!!

Reply:
It's ok for me to store a circle.. but when exporting, do not let the user export a point+radius!

Reply:
Can it be useful for certain exports format ? Maybe if someone wants a CSV file with the center coordinates and radius length ? If yes, i think the best would be to set default export to polygon, but allow the user to export a circle with radius if he really wants to

Reply:
The coordinates of the center (lon, lat) of a circle and its radius are kept as attributes for every format. Furthermore, the coordinates for point-type features are also saved as attributes. That way, CSVs contain more info and can be more useful in GIS applications.

Reply:
Good ! In that case, i would say always export as polygon